governance/program/src/state/enums.rs

```diff
@@ -152,11 +152,19 @@ pub enum VoteThreshold {
     /// In other words a '+1 vote' tie breaker is always required to have a successful vote
     YesVotePercentage(u8),
 
-    /// The minimum number of votes in % out of the entire pool of governance tokens eligible to vote
-    /// which must be cast for the vote to be valid
-    /// Once the quorum is achieved a simple majority (50%+1) of Yes votes is required for the vote to succeed
-    /// Note: Quorum is not implemented in the current version
-    QuorumPercentage(u8),
+    /// The minimum number of votes in basis points (1/10000) out of the entire pool of governance tokens eligible to vote
+    /// which must be cast for the proposal to be considered valid.
+    /// Once the quorum is achieved then pass level (in %) has to be casted for an option to succeed,
+    /// if defined to 50% it's a simple majority (50%+1) of Yes votes for the option to succeed
+    AttendanceQuorum {
+        #[allow(dead_code)]
+        /// The minimum number of votes in basis points (1/10000) out of the entire pool of governance tokens eligible to vote
+        /// which must be cast for the proposal being considered as successful
+        threshold: u16,
+        #[allow(dead_code)]
+        /// The minimum number of Yes votes in % required to be voted on an option for an option being successful
+        pass_level: u8,
```

Comment:
yes votes in % of what? entire pool of governance tokens? please update comment.

also "threshold" & "pass_level" could be more descriptive,
maybe just reuse previous terms "quorum_bps", "yes_vote_percentage"

Reply:
Correct, % of all governance tokens. Thanks for noting that. I tried to put it more descriptive.

I quite like the recommendation of used terms `quorum_bps` and `yes_vote_percentage`.